third_party/proxy/service.go

```diff
@@ -247,7 +247,7 @@ func (sct *ServiceChangeTracker) newBaseServiceInfo(port *v1.ServicePort, servic
 	// TODO: Switch to v1.DeprecatedAnnotationTopologyAwareHints and v1.AnnotationTopologyMode after
 	//  upgrading Antrea K8s API to at least 1.27
 	var ok bool
-	info.hintsAnnotation, ok = service.Annotations[v1.AnnotationTopologyAwareHints]
+	info.hintsAnnotation, ok = service.Annotations[v1.DeprecatedAnnotationTopologyAwareHints]
```

Comment:
Could you also handle above TODO and remove the TODO after you update `v1.AnnotationTopologyMode` as well? Thanks.

Reply:
Removed the `TODO` and replaced the hardcode annotation with `v1.AnnotationTopologyMode`.

Reply:
Please search `AnnotationTopologyAwareHints`, there are a few other places need to be updated. Thanks.